packages/dx-react-scheduler-material-ui/src/templates/layout.jsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
-import withStyles from '@mui/styles/withStyles';
+import { styled } from '@mui/material/styles';
 import * as PropTypes from 'prop-types';
 import { AUTO_HEIGHT } from '@devexpress/dx-scheduler-core';
-import { ContainerBase } from './common/container';
+import { ContainerBase, classes } from './common/container';
 
-const styles = {
-  container: {
+const StyledContainerBase = styled(ContainerBase)({
```

Comment:
Why we styled containerBase if it is not use classes?

Reply:
Replaced it to packages/dx-react-scheduler-material-ui/src/templates/common/container.jsx